.iex.exs

```diff
@@ -26,3 +26,6 @@ alias Aeutil.Parser
 alias Aeutil.Scientific
 alias Aeutil.Serialization
 alias Aeutil.PatriciaMerkleTree
+
+alias MerklePatriciaTree.Trie.Inspector
```

Comment:
why do you use this, instead of the PatriciaMerkleTree.pretty_print? I would prefer only aliasing our own components

Reply:
@meivantodorov 
Can you check this